### PR TITLE
Support aiven

### PR DESCRIPTION
**Pull request by fleetwoods**

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+w---
```
Comment (laurence-bird) [inline cloudformation.yaml]: minor: guessing this w isn't meant to be here

File: src/main/scala/com/ovoenergy/event/gateway/manager/kafka/SendEvent.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.ovoenergy.event.gateway.manager.kafka
```
Comment (laurence-bird) [inline src/main/scala/com/ovoenergy/event/gateway/manager/kafka/SendEvent.scala]: Think this isn't being used and can be deleted now

File: src/main/scala/com/ovoenergy/event/gateway/manager/sqs/Client.scala [hunk 3]
```diff
@@ -18,7 +18,7 @@ object Client extends LoggingWithMDC {
     val clientBuilder = AmazonSQSClientBuilder.standard()
     if (isRunningInCompose) {
       logInfo(Map(), "Building SQS Client for docker-compose")
-      clientBuilder.setEndpointConfiguration(new EndpointConfiguration("http://localhost:4100", "eu-west-1"))
+      clientBuilder.setEndpointConfiguration(new EndpointConfiguration("http://10.200.10.1:4100", "eu-west-1"))
```
Comment (tommpy) [inline src/main/scala/com/ovoenergy/event/gateway/manager/sqs/Client.scala]: Eh??

File: src/servicetest/scala/servicetest/DockerIntegrationTest.scala [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+
+  override implicit lazy val dockerExecutionContext: ExecutionContext = {
+    // using Math.max to prevent unexpected zero length of docker containers
+    ExecutionContext.fromExecutor(Executors.newFixedThreadPool(Math.max(1, dockerContainers.length * 4)))
+  }
+
+  val hostIpAddress = {
+    import sys.process._
+    "./get_ip_address.sh".!!.trim
+    //"10.200.10.1" // TODO revert me
```
Comment (tommpy) [inline src/servicetest/scala/servicetest/DockerIntegrationTest.scala]: revert me?

File: src/servicetest/scala/servicetest/DockerIntegrationTest.scala [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+  lazy val dynamodb = DockerContainer("forty8bit/dynamodb-local:latest", name = Some("dynamodb"))
+    .withPorts(8000 -> Some(8000))
+    .withCommand("-sharedDb")
+    //    .withCommand("java",
+    //                 "-Xmx256M",
+    //                 "-Xms128M",
+    //                 "-Djava.library.path=./DynamoDBLocal_lib",
+    //                 "-jar",
+    //                 "DynamoDBLocal.jar",
+    //                 "-sharedDb") // TODO we need to override the entrypoint, not pass a command. This will need a PR against docker-it-scala.
```
Comment (tommpy) [inline src/servicetest/scala/servicetest/DockerIntegrationTest.scala]: Can these be deleted?

File: src/test/resources/logback.xml [hunk 6]
```diff
@@ -8,7 +8,7 @@
         </encoder>
     </appender>
 
-    <root level="WARN">
+    <root level="INFO">
```
Comment (tommpy) [inline src/test/resources/logback.xml]: Does this need to go back to WARN?

File: README.md [hunk 7]
```diff
@@ -37,7 +37,7 @@ The docker-compose.yml file included in the project is used for service testing
 
 ### Unit Test
 
-Tests are executed via `sbt test`
+Tests are executed via `sbt testWithAws`
 
 ### Service Tests
```
Comment (cb372) [inline README.md]: could you update the section about using `sbt dockerComposeTest`

File: cloudformation.yaml [hunk 8]
```diff
@@ -122,7 +122,7 @@ Resources:
               - sts:AssumeRole
       Path: /
       Policies:
-        -
```
Comment (cb372) [inline cloudformation.yaml]: Look like you still need to add the permission to download the keystore and truststore

File: project/Dependencies.scala [hunk 9]
```diff
@@ -1,28 +1,34 @@
 import sbt._
 
 object Dependencies {
-  val kafkaMessagesVersion = "0.0.14"
-  val akkaHttpVersion      = "10.0.2"
+  val kafkaMessagesVersion           = "1.25"
+  val kafkaSerialisationVersion      = "2.6"
```
Comment (cb372) [inline project/Dependencies.scala]: may as well bump this to 2.8 before merging this

File: project/Dependencies.scala [hunk 10]
```diff
@@ -1,28 +1,34 @@
...
+  val akkaHttpVersion                = "10.0.2"
+  val commsKafkaSerialisationVersion = "2.7"
```
Comment (cb372) [inline project/Dependencies.scala]: Actually I meant this line. Looks like the `val` I commented on above is not used and can be deleted

File: project/plugins.sbt [hunk 11]
```diff
@@ -1,5 +1,4 @@
-addSbtPlugin("com.geirsson" % "sbt-scalafmt"       % "0.5.6")
-addSbtPlugin("com.tapad"    % "sbt-docker-compose" % "1.0.19")
+addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "0.5.6")
 
 resolvers += Resolver.bintrayIvyRepo("ovotech", "sbt-plugins")
 addSbtPlugin("com.ovoenergy" % "sbt-comms-packaging" % "0.0.7")
```
Comment (cb372) [inline project/plugins.sbt]: This needs to be bumped. Latest version is 0.0.10.

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 196](https://circleci.com/gh/ovotech/comms-gateway-event-manager/196) at 2017-07-06 14:42 BST.

**Pull request merged**